extensions/amp-bodymovin-animation/0.1/amp-bodymovin-animation.js

```diff
@@ -72,10 +81,7 @@ export class AmpBodymovinAnimation extends AMP.BaseElement {
    */
   preconnectCallback(opt_onLayout) {
     const preconnect = Services.preconnectFor(this.win);
-    const scriptToLoad =
-      this.renderer_ === 'svg'
-        ? 'https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin_light.min.js'
-        : 'https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin.min.js';
+    const scriptToLoad = libSourceUrl[this.renderer_] ?? libSourceUrl.svg;
     preloadBootstrap(this.win, this.getAmpDoc(), preconnect);
     preconnect.url(this.getAmpDoc(), scriptToLoad, opt_onLayout);
```

Comment:
[`<link rel=preconnect>` only cares about origin](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preconnect). I'm not sure why this specified distinct paths in the first place.

It's probably better to simply connect to that origin but make it clear what's going on. 

```suggestion
    preloadBootstrap(this.win, this.getAmpDoc(), preconnect);
    // Different scripts are loaded based on `renderer` but their origin is the same.
    // See 3p/bodymovinanimation.js # libSourceUrl
    preconnect.url(
      this.getAmpDoc(),
      "https://cdnjs.cloudflare.com",
      opt_onLayout
    );
```